modules/installation-configuration-parameters.adoc

```diff
@@ -252,6 +252,7 @@ ifdef::osp[]
 |`machines.platform.openstack.defaultMachinePlatform`
 | _Optional_. The default configuration used when installing on OpenStack for
 machine pools that do not define their own platform configuration.
+//This remains in question. Resolve before merging new PR.
```

Comment:
@tomassedovic @iamemilio Is there an answer to this? It was up in the air at the time of writing this module.

Reply:
All good here.